app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,4 @@
+class CustomerItemsController < ApplicationController
```

Comment:
You were totally on the right track here with a joins controller! Since you are adding an item to a customer, what you're doing on a data level is creating a joins record in the customer_items table. So, you can have a `create` action here to create that record, and then redirect back to the customer show page